libs/execution_context/ebpf_native.c

```diff
@@ -1195,6 +1224,90 @@ _ebpf_native_get_count_of_programs(_In_ const ebpf_native_module_t* module)
     return count_of_programs;
 }
 
+static void
+_ebpf_native_close_handles_workitem(_In_opt_ const void* context)
+{
+    if (context == NULL) {
+        return;
+    }
+
+    ebpf_native_handle_cleanup_information_t* handle_info = (ebpf_native_handle_cleanup_information_t*)context;
```

Comment:
```suggestion
    ebpf_native_handle_cleanup_info_t* handle_info = (ebpf_native_handle_cleanup_information_t*)context;
```
be consistent between variable name and type name.
(And `info` is ok since it's in the dictionary (https://www.dictionary.com/browse/info) so doesn't count as an abbreviation)